e2e/subgraph/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export function reverseString(value: string): string {
+  return value;
```

Comment:
I would expect this to be reversed.

Reply:
<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia3.giphy.com%2Fmedia%2F5d5VxTUXYJiWQ%2Fgiphy-downsized-medium.gif&token=catch" data-gitmeme-token="catch"><img src="https://media3.giphy.com/media/5d5VxTUXYJiWQ/giphy-downsized-medium.gif" title="Created by gitme.me with /catch"/></a>